### DISCUSSION TITLE
Translations in images

### DISCUSSION THREAD

**Discussion by manuxi** [sulu/sulu#7629] [Ideas]
Would be great to have a possibility to copy the translation of a locale as it is possible in pages. I have about 600 images here. Would be great to speed up the process of adding info to them.

**Comment by alexander-schranz** [comment]
Media normally don't require a copy language as they alwas fallback to the default locale. Lets have a look at the following media which was just created in EN:

![Bildschirmfoto 2024-10-29 um 15 23 05](https://github.com/user-attachments/assets/40e024ff-ff0a-484f-be43-a237695b817d)

As Media has a fallback mechanism and you switch to DE you directly get the media in the default locale:

![Bildschirmfoto 2024-10-29 um 15 23 13](https://github.com/user-attachments/assets/13fd8e4f-8aff-4614-9857-48c2247f9cdc)

You can then change and do your adoption. Still I think it may make sense to have copy locale for later usage, but keep this in mind that there is if the media was never saved in a language a fallback mechanism.

**Comment by manuxi** [comment]
Probably the problem was as follows: I uploaded the images in en and they got the titles from the image names. I then switched to 'de' and modified the images titles, descriptions, and so on. Then en wasn't touched and I had to do that again. Will keep that in mind for next time. Wasn't aware which locale my fallback was...
